template/src/style.scss

```diff
@@ -4,11 +4,11 @@
 // ---------------------------------------------------------------------
 
 // Import Auro tokens
-@import './node_modules/@alaskaairux/design-tokens/dist/tokens/SCSSVariables';
-@import './node_modules/@alaskaairux/design-tokens/dist/tokens/SassCustomProperties';
+@import './../node_modules/@alaskaairux/design-tokens/dist/tokens/SCSSVariables';
```

Comment:
This path update was not applied to the demo style.scss and breaks the build when generating a new CE repo.